src/Examples/Advanced/CustomUnitModels/methanol_state_block_VLE.py

```diff
@@ -299,7 +306,9 @@ def rule_comp_mass_balance(self, i):
     def _make_flash_eq(self):
         self.vapor_pressure = Var(self._params.component_list,
                                   initialize=0.101325,
-                                  doc="vapor pressure ",
+                                  domain=NonNegativeReals,
+                                  units=pyunits.MPa, 
+                                  doc="vapor pressure [MPa]",
```

Comment:
Typo: doc string should start with a capital. I also tend not to include units in the doc string when they are explicitly defined.

Reply:
I'd like the `doc` tags on lines 254, 258, 311, and 354 to be consistent. Would you recommend removing the units from all of them, as well as line 127 in the the main param_VLE file?

Reply:
Doc strings should be consistent. Others might disagree with not including units in the doc string however - I personally prefer not including them to avoid any chance of a mismatch between the docs and the defined units.